pandas/core/series.py

```diff
@@ -1601,6 +1601,7 @@ def __repr__(self) -> str:
         """
         Return a string representation for a particular Series.
         """
+        # pylint: disable=invalid-repr-returned
```

Comment:
this'll turn it off for the rest of the file - can you either turn it off inline, or address the issue that's causing the error?

Reply:
pylint doc says this will disable it for single scope . 

https://pylint.pycqa.org/en/latest/user_guide/messages/message_control.html#block-disables

Also, to be sure, I have add a dummy class with an invalid `__repr__` at the bottom of the file and it highlighted the issue. 
![image](https://user-images.githubusercontent.com/40605798/194859786-c5cfd5bb-a375-4776-9a6a-3108943b90d4.png)